redis/src/script.rs

```diff
@@ -213,6 +213,15 @@ impl<'a> ScriptInvocation<'a> {
     }
 
     fn eval_cmd(&self) -> Cmd {
+        let args_len = 3 + self.keys.len() + self.args.len();
+        let buf_len = self
+            .keys
+            .iter()
+            .chain(self.args.iter())
+            .fold(0, |acc, e| acc + e.len())
+            + 7 /* "EVALSHA".len() */
+            + self.script.hash.len();
+        let mut cmd = Cmd::with_capacity(args_len, buf_len);
         let mut cmd = cmd("EVALSHA");
```

Comment:
this overwrites the previous `cmd`, and you've essentially done nothing. you'll need to replace this with `arg`.

Reply:
0.0..

Sorry, it's a mistake...